### PR TITLE
Helm Chart

### PR DESCRIPTION
**Pull request by uwe-klinger**

### CODE REVIEW

File: srv/pom.xml [hunk 1]
```diff
@@ -104,6 +120,19 @@
...
+        </dependency>
+        <dependency>
+            <groupId>com.sap.cloud.security.xsuaa</groupId>
+            <artifactId>xsuaa-spring-boot-starter</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.sap.cds</groupId>
+            <artifactId>cds-feature-hana</artifactId>
+        </dependency>
```
Comment (beckermarc) [inline srv/pom.xml:135]: Those dependencies automatically come with cds-starter-cloudfoundry. No need to add them here..

File: srv/src/main/java/my/bookshop/config/DestinationConfiguration.java [hunk 2]
```diff
@@ -20,13 +21,16 @@ public class DestinationConfiguration {
...
+		Boolean onPremise = environment.getProperty("on.premise", Boolean.class);
 		if(port != null && destinationName != null) {
 			DefaultHttpDestination httpDestination = DefaultHttpDestination
 			.builder("http://localhost:" + port)
 			.name(destinationName).build();
 
 			DestinationAccessor.prependDestinationLoader(
 				new DefaultDestinationLoader().registerDestination(httpDestination));
+		} else if (destinationName == "s4-destination" && onPremise) {
+			DestinationAccessor.getDestination("s4-destination").asHttp().decorate(DefaultErpHttpDestination::new);
```
Comment (beckermarc) [inline srv/src/main/java/my/bookshop/config/DestinationConfiguration.java:33]: I think this doesn't work. There is currently a workaround required for on-premise destinations, which is described here: https://github.wdf.sap.corp/MA/cap-cloud-sdk-collaboration/issues/55#issuecomment-2948646

File: srv/pom.xml (hunk 1 above)
Comment (uwe-klinger) [reply srv/pom.xml:135]: Ok, I'll remove them and retry.

File: srv/src/main/java/my/bookshop/config/DestinationConfiguration.java (hunk 2 above)
Comment (uwe-klinger) [reply srv/src/main/java/my/bookshop/config/DestinationConfiguration.java]: As discused, the code doesn't work.  The header field must be configured in the destination.

File: srv/pom.xml [hunk 3]
```diff
@@ -61,6 +67,16 @@
 			<artifactId>scp-cf</artifactId>
 		</dependency>
 
+		<!-- CLOUD SDK for on-premise connectivity -->
```
Comment (uwe-klinger) [inline srv/pom.xml:70]: @beckermarc: I wonder if we still need it. I could retry without.

Comment (beckermarc) [reply srv/pom.xml:70]: I guess we do, if you want to connect to on premise systems. But you can try out what happens without these dependencies. AFAIK they are however responsible for implementing the connectivity service stuff, which is not required for cloud S/4 hana systems. Not sure if the setup to connect to an on-premise S/4 needs to be part of the sample though.

**Review by beckermarc — APPROVED** [review]

**Pull request merged**